testing/common/foreignkey_test.go

```diff
@@ -0,0 +1,206 @@
+// Copyright 2020 Google LLC
```

Comment:
the location for this new test is a bit odd, this top level testing/ subdirectory is for integration tests, whereas as I understand foreign key tests are unit testing a specific piece of the codebase

suggest moving these tests to conversion/conversion_test.go for now (which is kind of unfortunate and we are forced to do that since conversion.go has become a dumping ground for all sort of unrelated functions) -- but the hope is that once the UpdateDDLForeignKeys function moves to a more appropriate module, then these tests would also move out from conversion_test.go